common/ipmi/storage_handler.c

```diff
@@ -0,0 +1,225 @@
+#include "storage_handler.h"
+#include "plat_fru.h"
+#include "fru.h"
+#include "sdr.h"
+
+__weak void STORAGE_GET_FRUID_INFO(ipmi_msg *msg)
+{
+	uint8_t fruid;
+	uint16_t fru_size;
+
+	if (msg->data_len != 1) {
```

Comment:
msg == NULL check?

Reply:
We have fixed it and done the forced update.
Please review it. Thanks.